src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/RecordMappers.kt

```diff
@@ -76,7 +76,8 @@ class RecordMappers {
                     record.getValue(RESOURCE_LINK.ID),
                     record.getValue(RESOURCE_LINK.RESOURCE_CONTENT_FK),
                     record.getValue(RESOURCE_LINK.CONTENT_FK),
-                    record.getValue(RESOURCE_LINK.COLLECTION_FK)
+                    record.getValue(RESOURCE_LINK.COLLECTION_FK),
+                    record.getValue(RESOURCE_LINK.RC_FK)
```

Comment:
I hate to do this right now.  I know we've already discussed this.  But...

Would it be better for this to be in the Content table?  That would mean that selecting all the content for a given location and RC would use the same process whether it's a resource or main text.  Done this way, with rcfk in the resource table, you'd join with that table in the case of resources, but join with parent collection in the case of main text.

Reply:
I might have to come back to this tomorrow, but I am willing to revisit it